src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -926,7 +919,7 @@ private void InitializeNodeLibrary(IPreferences preferences)
             CustomNodeManager.AddUninitializedCustomNodesInPath(pathManager.CommonDefinitions, IsTestMode);
         }
 
-        private void LoadNodeLibrary(Assembly assem)
+        public void LoadNodeLibrary(Assembly assem)
```

Comment:
Instead of making this public, let's make it `internal` and create a small wrapper class around it implement `ILibraryLoader`.